packages/contracts/contracts/voting/erc20/ERC20Voting.sol

```diff
@@ -75,21 +78,21 @@ contract ERC20Voting is MajorityVoting {
 
         voteId = votesLength++;
 
-        // calculate start and end time for the vote
+        // Calculate the start and end time of the vote
         uint64 currentTimestamp = getTimestamp64();
 
         if (_startDate == 0) _startDate = currentTimestamp;
         if (_endDate == 0) _endDate = _startDate + minDuration;
 
         if (_endDate - _startDate < minDuration || _startDate < currentTimestamp)
-            revert VoteTimesForbidden({
+            revert VoteTimesInvalid({
```

Comment:
```suggestion
            revert InvalidVoteTimes({
```

Reply:
When changing this, we also should change the order in the other errors
```solidity
error VoteSupportExceeded(uint64 limit, uint64 actual);
error VoteParticipationExceeded(uint64 limit, uint64 actual);
error VoteDurationZero();
error VoteCastingForbidden(uint256 voteId, address sender);
error VoteExecutionForbidden(uint256 voteId);
```
I don't see an improvement here and would keep it the same way as in the rest of the repository